### PR TITLE
[PM-3294][PM-3293] Update Region Selector Domains

### PR DESCRIPTION
**Pull request by rr-bw**
## Type of change

- Tech debt (refactoring, code cleanup, dependency upgrades, etc)

## Objective

On web clients, updates text of region selector to show:
- "bitwarden.com" instead of "US"
- "bitwarden.eu" instead of "EU"

Updates labels of region selector to show:
- "Logging in on:" (Desktop, Browser)
- "Server:" (Web)

## Code changes

- Update translations
    - `apps/browser/src/_locales/en/messages.json`
    - `apps/web/src/locales/en/messages.json`
    - `apps/desktop/src/locales/en/messages.json`
- Update text interpolation
    - `apps/web/src/app/components/environment-selector/environment-selector.component.html`
    - `libs/angular/src/auth/components/environment-selector.component.html`

### Desktop
Screenshot 2023-08-04 at 11 19 36 AM

### Browser
Screenshot 2023-08-04 at 11 30 12 AM

### Web
Screenshot 2023-08-03 at 9 38 11 AM

### Trial Initiation Header
Screenshot 2023-08-04 at 1 17 43 PM

### CODE REVIEW

File: apps/desktop/src/locales/en/messages.json [hunk 1]
```diff
@@ -2252,16 +2252,14 @@
   "windowsBiometricUpdateWarningTitle": {
     "message": "Recommended Settings Update"
   },
-  "region": {
-    "message": "Region"
+  "loggingInOn": {
+    "message": "Logging in on"
```
Comment (differsthecat) [inline apps/desktop/src/locales/en/messages.json:2256]: Does this need to be added to the web messages.json too? It appears to be used by the `environment-selector` component In libs, and then used in web for the `trial-initiation` and `frontend-layout` components.

Comment (trmartin4) [reply apps/desktop/src/locales/en/messages.json:2256]: The web client should use the "server" localization, whereas the other clients should use "loggingInOn" (the story asked for different verbiage on web).  It looks like "server" is already in the `messages.json` for web, which is why it doesn't show up in the diff here.

That being said, could we get rid of the "region" phrase now?

Comment (rr-bw) [reply apps/desktop/src/locales/en/messages.json:2256]: @trmartin4 Yes I'll remove that.

**Review by trmartin4 — DISMISSED** [review]
Looks good!  Thank you!

**Review by trmartin4 — APPROVED** [review]
Thank you for making the change to align "Self-hosted".  Looks good!

**Review by trmartin4 — APPROVED** [review]

**Pull request merged**